src/com/amazon/corretto/crypto/provider/EvpDhKey.java

```diff
@@ -0,0 +1,28 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.amazon.corretto.crypto.provider;
+
+import javax.crypto.interfaces.DHKey;
+import javax.crypto.spec.DHParameterSpec;
+
+
+abstract class EvpDhKey extends EvpKey implements DHKey {
+    private static final long serialVersionUID = 1;
```

Comment:
Nitpick -- to avoid a cast on `long` literals you can use `1L`.

Reply:
I'm not sure why this would matter as it is evaluated at compile time and stored in a constant.